community/open-meetings.mdx

```diff
@@ -58,6 +58,14 @@ These are dedicated sync meetings for specific products, as well as open plannin
              }
 />
 
+<MeetingInfo title="[Portal] Portal Open Meeting"
+             schedule="Every Thursday effective 4. Jul 2024 until 31. Dec 2024 from 01:30 pm to 02:00 pm CEST"
+             description="Coordination of feature refinement and development for portal."
+             contact="erik.wiegert@cofinity-x.com"
+             sessionLink="https://teams.microsoft.com/l/meetup-join/19%3ameeting_YmIyYjgwOTQtZjI1Ny00YmM0LTlmOWQtODRjZWFmZWM1Y2E3%40thread.v2/0?context=%7b%22Tid%22%3a%221ad22c6d-2f08-4f05-a0ba-e17f6ce88380%22%2c%22Oid%22%3a%22a8b7a5ee-66ff-4695-afa2-08f893d8aaf6%22%7d"
+             meetingLink="/meetings/portal-open-meeting.ics"
+/>
+
```

Comment:
would you great if we could also link the matrix chat https://chat.eclipse.org/#/room/#tractusx-portal:matrix.eclipse.org

Reply:
added additional links. thx